src/seedu/addressbook/data/person/Address.java

```diff
@@ -55,4 +56,11 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+	@Override
+	public String getPrintableString() {
+	    // TODO Auto-generated method stub
```

Comment:
Please delete this because you have already created the method. It is no longer TODO.